### DISCUSSION TITLE
Espresso Timer only showing "pouring" time instead of the whole shot time

### DISCUSSION THREAD

**Discussion by christophedc0** [obiwan007/despresso#66] [Beta]
It might be on purpose, but for now the espresso timer only shows the pouring time.

I am used to seeing the complete shot time from the moment when I press the button to start the shot.

for reference, the complete shot time can be seen in the graphs on the bottom of the temperature graph.

![IMG_2149](https://user-images.githubusercontent.com/62201948/224537956-796ded8d-e07b-4273-b401-53a83d45361b.jpeg)

**Comment by obiwan007** [comment]
This will be fixed in the next release
![image](https://user-images.githubusercontent.com/12936285/224539798-f52e0eda-3ee6-46f4-b54f-2a785d5f55c1.png)
